### DISCUSSION TITLE
I don't quite understand how to deploy plain nodejs.

### DISCUSSION THREAD

**Discussion by Izzat32** [vercel/community#1588] [Help] [answered]
### Summary

Is it possible? I only see next.js and nuxt.js and other stuff.

**Comment by leerob** [comment, marked as answer]
https://github.com/vercel/examples/tree/main/solutions/node-hello-world

**Reply by amyegan** [reply]
That example is a great place to start! And you can find more info about the Node.js Runtime here: [Using the Node.js Runtime with Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#)
